### DISCUSSION TITLE
第3节课作业(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#357] [作业]
第3节课作业(24班)

**Comment by williamleo0730** [comment]
https://zhuanlan.zhihu.com/p/677986488

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/47b1ea16-939c-400c-902c-e191a872ac83

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135575667

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day3_0.md

**Comment by waurblove** [comment]
https://github.com/waurblove/LLM/blob/main/day3.md

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135717827?spm=1001.2014.3001.5501

**Comment by PopoData** [comment]
https://zhuanlan.zhihu.com/p/678905996

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29832431/

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135722099?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135721854?spm=1001.2014.3001.5501

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/678919553

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135729103?spm=1001.2014.3001.5502
